Cross.toml

```diff
@@ -1,8 +1,14 @@
 [build]
-default-target = "x86_64-unknown-linux-gnu"
+default-target = "x86_64-unknown-linux-musl"
```

Comment:
I think we should keep the default target to gnu, rather than musl unless we have some strong reason not to...

Reply:
This only impacts when cross is called without specifying a target. The cargo.sh script always specifies a target, so it doesn't use the default.

I'll change it to gnu for consistency.